rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/Util.rsc

```diff
@@ -108,6 +108,13 @@ bool isShorterTuple(tuple[loc, &T] t1, tuple[loc, &T] t2) = isShorter(t1[0], t2[
 }
 bool byOffset(loc l1, loc l2) = l1.offset < l2.offset;
 
+@synopsis{
+    Gives the longest location in a non-empty set of locations.
+    On a tie, gives one of the longest locations.
+}
+loc maxByLength(set[loc] locs:{_, *_}) = // at least one element
```

Comment:
nit: if there is at least one, that can also be used as the initial case (instead of an `unkown` loc).

```rascal
loc maxByLength({loc atLeastOne, set[loc] *rest}) 
    = (atLeastOne | it.length > l.length ? it : l | l.locs);
```

Reply:
This is actually unused. Added it for an earlier implementation, and then rewrote. Will remove!